src/App.tsx

```diff
@@ -1,10 +1,50 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './TodoList';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { TodoFilter } from './TodoFilter';
 
-const USER_ID = 0;
+const USER_ID = 10603;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState('all');
```

Comment:
Let's create an enum for that so you won't have to use simple strings in the code
```suggestion
  const [filterType, setFilterType] = useState(FilterType.ALL);
```